src/coreimage.cs

```diff
@@ -3385,11 +3580,15 @@ interface CIFalseColor {
 
 	[CoreImageFilter]
 	[BaseType (typeof (CITransitionFilter))]
-	interface CIFlashTransition {
+	interface CIFlashTransition : ICIFlashTransitionProtocol {
 
+#if !XAMCORE_4_0
 		// for some reason we prefixed all Striation* with Max - API compatibility
 		[CoreImageFilterProperty ("inputStriationContrast")]
```

Comment:
```suggestion
		[Obsolete ("Use 'StriationContrast' instead.")]
		[CoreImageFilterProperty ("inputStriationContrast")]
```

Reply:
noise ;-)

Reply:
Yea, read your description and forgot about it after 10k lines reviewed :P